### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by samwel-mongare**
### The branch styling has the following features to be merged into the development branch:

- Added basic styling to the app.
- Implemented the use of react icons
- Used CircularProgressbar from react-circular-progressbar to show book progress

### CODE REVIEW

File: src/components/BooksUI/Bookinput.js [hunk 1]
```diff
@@ -41,17 +41,18 @@ const Bookinput = (() => {
...
         <option value="Action">Action</option>
         <option value="Horror">Horror</option>
         <option value="Science Fiction">Science Fiction</option>
         <option value="Economy">Economy</option>
         <option value="Comedy">Comedy</option>
       </select>
-      <button type="submit">Add</button>
+      <button type="submit" className="submit">Add Book</button>
     </form>
   );
```
Comment (Emmyn5600) [inline src/components/BooksUI/Bookinput.js:57]: - [ ] Kindly, compare your implementation with the design provided to meet all the requirements. Please use the same font size, color, and font family.

Original design

header--2

Your design

header--6

File: src/components/Navbar.js [hunk 2]
```diff
@@ -49,6 +27,9 @@ const Navbar = () => {
           </li>
         ))}
       </ul>
+      <div className="holder">
+        <FaUserCircle className="user" />
+      </div>
```
Comment (Emmyn5600) [inline src/components/Navbar.js:32]: - [ ] Kindly, compare your implementation with the design provided to meet all the requirements. Please use the same font size, color, and font family.

Original design.

header-2

Your design

header--5

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @Mosams 

Great work on making the project requirements 👏 
You've done well implementing some of the project requirements, but there are still some which aren't addressed yet, please work on it and ask for another review.

 ## Required Changes ♻️
- Check the comments under the review.

## Optional suggestions

- n/a

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/components/Navbar.js (hunk 2 above)
Comment (samwel-mongare) [reply src/components/Navbar.js:32]: Hello, I have tried my best to match the colors, font size, and font-weight. Different screens, however, have different resolutions and color densities which might affect the final output. Please take that into consideration. I have copied everything from the Zeplin CSS section and will continue perfecting the interface and functionality of the app. Thank you for the review.

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉 

## Hi @Mosams, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional
- I suggest that you add some padding between booklist and a form, currently they are overlapping
pad



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**